### PR TITLE
Publish release over Sonatype repository.

### LINKED ISSUE #216: Bintray Download for 6.1.0 down
As we had some problems very similar to #214 we updated play-ebean to 6.1.0-RC3 with the following line in `plugins.sbt`:
`addSbtPlugin("com.typesafe.sbt" % "sbt-play-ebean" % "6.1.0-RC3")`

This worked fine but today I experimented with building in a docker container and then got the exception that play-ebean couldn't be loaded as `https://dl.bintray.com/playframework/maven/com/typesafe/play/play-ebean_2.13/6.1.0-RC3/play-ebean_2.13-6.1.0-RC3.pom` returned a 403. And in fact, the complete artifact directory at https://dl.bintray.com/playframework/maven/com/typesafe/play/play-ebean_2.13/6.1.0-RC3 returns a 403. 

I'm not enough into maven, bintray etc. so this is purely a speculation but could this be related to the [bintray sunset](https://www.jfrog.com/confluence/display/BT/Welcome+to+JFrog+Bintray)? According to this link the community central was shut down on May 1st. But maybe this ist just temporary, I don't know.

It looks like the 6.0.0 is hosted directly on maven central so it isn't affected. But as the ebean version changed I fear that downgrading would produce a lot of errors in my project, so is there a way to set a mirror for downloading play-ebean? Or could a contributor move the artifacts away from bintray?

### PR DESCRIPTION
**Pull request by felipebonezi**
Fixes #216

### CODE REVIEW

**Comment by felipebonezi** [comment]
@octonato check this out! 😄
Is everything right?

File: .travis.yml [hunk 1]
```diff
@@ -7,18 +7,23 @@ env:
...
 
 stages:
   - name: validate
   - name: docs
   - name: jacoco
   - name: test
-  - name: deploy
-    if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork
+  - name: release
+    if: tag IS present
```
Comment (octonato) [inline .travis.yml:26]: I think we can keep the `if` condition. That is for publishing snapshots once PR are merged in master.

File: sbt-play-ebean/src/sbt-test/sbt-ebean/enhancement/build.sbt [hunk 2]
```diff
@@ -13,6 +13,3 @@ javaSource in Test := baseDirectory.value / "tests"
 resolvers ++= DefaultOptions.resolvers(snapshot = true)
 
 libraryDependencies += "com.h2database" % "h2" % "1.4.196"
-
-resolvers += "scalaz-releases" at "https://dl.bintray.com/scalaz/releases" // specs2 depends on scalaz-stream
```
Comment (octonato) [inline sbt-play-ebean/src/sbt-test/sbt-ebean/enhancement/build.sbt:17]: ???

**Comment by octonato** [comment]
@felipebonezi, sorry I clicked to fast on the button and sent the review without comments.

It's looking good, but we may want to preserve the same `if` condition in the Travis build. 

I will add the credentials for Sonatype to Travis

**Comment by octonato** [comment]
@felipebonezi, secrets added to travis. 

If you change the `if` condition we can merge this and the snapshot will be published. Assuming I added all credentials correctly.

File: .travis.yml (hunk 1 above)
Comment (felipebonezi) [reply .travis.yml:26]: OK!

Comment (octonato) [inline .travis.yml:26]:
```suggestion
    if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork
```

**Comment by octonato** [comment]
@felipebonezi, I applied the changes and will merge. Hopefully the snapshot gets published confirming that we setup everything correctly.

**Comment by skiley22** [comment]
The artifacts were not successfully published: https://app.travis-ci.com/github/playframework/play-ebean/jobs/510825770

**Comment by octonato** [comment]
@skiley22, yes. This is known. We had similar issues in other Play project. The fix is explained here: https://github.com/playframework/play-ebean/pull/219#issuecomment-880930591

**Comment by felipebonezi** [comment]
> @skiley22, yes. This is known. We had similar issues in other Play project. The fix is explained here: [#219 (comment)](https://github.com/playframework/play-ebean/pull/219#issuecomment-880930591)

I've fixed on PR #222 
Could you review? 😄

**Comment by felipebonezi** [comment]
> The artifacts were not successfully published: https://app.travis-ci.com/github/playframework/play-ebean/jobs/510825770

Fixed

https://app.travis-ci.com/github/playframework/play-ebean/jobs/540030893

**Pull request merged**